### PR TITLE
README: update to cmake

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,22 +202,20 @@ The executable can be found in the build/release/bin folder.
...
 
-8. Start the build
+6. Start the build
 
-  `./build.sh`
+    ```
+    make release -j4
+    ```
+    \* `4` - number of CPU threads to use  
+    \* Add e.g. `CMAKE_PREFIX_PATH=$HOME/Qt/5.9.7/clang_64/bin` enviroment variable to set a custom Qt install directory
```
Comment (xiphon) [inline README.md:218]: The path should not include `/bin`

Comment (selsta) [reply README.md:218]: Yep, copy paste error

**Review by xiphon — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -173,14 +173,18 @@ The following instructions will fetch Qt from your distribution's repositories i
...
 
 4. Build
 
     ```
-    cd monero-gui
-    QT_SELECT=5 ./build.sh
+    make release -j4
     ```
+    \* `4` - number of CPU threads to use  
+    \* Add e.g. `CMAKE_PREFIX_PATH=$HOME/Qt/5.9.7/gcc_64/` enviroment variable to set a custom Qt install directory
```
Comment (xiphon) [inline README.md:187]: Could we slightly rephrase this to indicate that `CMAKE_PREFIX_PATH` should be prepended to `make` command. Or do you think the current wording is clear enough?

Comment (selsta) [reply README.md:187]: Maybe like this?

```suggestion
    \* Add e.g. `CMAKE_PREFIX_PATH` enviroment variable to set a custom Qt install directory, e.g. `CMAKE_PREFIX_PATH=$HOME/Qt/5.9.7/gcc_64/  make release -j4`
```

Or do you have better wording?

Comment (xiphon) [reply README.md:187]: Looks good

**Review by xiphon — APPROVED** [review]
